ckanext/geocat/harvester.py

```diff
@@ -217,6 +218,8 @@ def import_stage(self, harvest_object):  # noqa
                         'NonCommercialNotAllowed-CommercialNotAllowed-ReferenceRequired'  # noqa
                     )
 
+            swisstopo_relation = \
```

Comment:
Can you explain to me why it is called swisstopo_relation ? if it only applies to swisstopo I am not sure that it belongs in the geocat-harvester-base.
But I guess it is an optional field that swisstopo already provides and possible other sources might not? In that case it would be great if you consider naming it differently.